weld/src/main/java/org/jboss/as/weld/WeldExtension.java

```diff
@@ -61,7 +61,7 @@
 
     private static final String RESOURCE_NAME = WeldExtension.class.getPackage().getName() + ".LocalDescriptions";
 
-    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(2, 0, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(2, 1, 0);
```

Comment:
given that this will be part of 10.x series not 9.x, version should be 3.0.0 
2.x.x are reserved for 9.x releases if there might be some more.